external/test-runtime/optional.json

```diff
@@ -0,0 +1,26 @@
+{
+  "frameworks": {
+    "net45": {
+      "dependencies": {
+        "Microsoft.DotNet.IBCMerge": "4.6.0-alpha-00001",
+        "Microsoft.DotNet.UAP.TestTools": "1.0.2",
+        "TestILCNugetPackageForCoreFX": "1.0.0-beta-25131-00"
+      }
+    }
+  },
+  "runtimes": {
```

Comment:
Do we actually need all the RIDs? Do we have any native assets in these packages?